demo-app/src/app/toast/toast.component.html

```diff
@@ -0,0 +1,117 @@
+<h3>Toast</h3>
+
+<p>
+  Materialize provides a toast implementation that will automatically be placed and sized according to the screen size of your device. Toasts are triggered programmaticaly using the service <code class="language-markup">ToastService</code> and can be customized through the parameters of the method <code class="language-markup">show</code>.
```

Comment:
according to the ~~screen size of your device~~. -> according to the device's screen resolution.